src/components/AccountDrawer/AuthenticatedHeader.tsx

```diff
@@ -135,7 +135,6 @@ const HeaderWrapper = styled.div`
 
 const CopyText = styled(CopyHelper).attrs({
   InitialIcon: Copy,
-  CopiedIcon: Copy,
   gap: 4,
```

Comment:
Minor nit: `CopyHelper` already has the same `CopiedIcon` and `gap` styles so those can be removed as well.

Reply:
fixed!